### DISCUSSION TITLE
Clarifying the `NN` Suffix: Defining Newtonian Nudging in `source_id` Naming

### DISCUSSION THREAD

**Discussion by csteger** [WCRP-CORDEX/discuss#13] [General]
### label

ICON-CLM-202407-1-1

### label_extended

ICON model in Climate Limited-area Mode (ICON-CLM), Version 202407-1-1

### source_id

ICON-CLM-202407-1-1

### source_type

ARCM

### release_year

2024

### activity_participation

DD

### institution_id

CLMcom-DWD

### further_info_url

https://www.clm-community.eu/ICON-CLM/

### What license do you choose?

CC BY 4.0

**Comment by csteger** [comment]
We will add more instition_ids when they are registered. 

I was in contact with Grisha, because we are not sure about the NN suffix for the nudging. He suggested to open an issue here to discuss it. 

The archiving specification document says, that SN or NN should be added to the source_id for spectral or Newtonian nudging. Does Newtonian nudging here mean Newtonian relaxation at the boundaries or nuding in the entire domain? In addition, in ICON-CLM we apply an “upper boundary nudging” (see section 6.2 of the ICON tutorial: https://www.dwd.de/DE/leistungen/nwv_icon_tutorial/pdf_einzelbaende/icon_tutorial2024.pdf;jsessionid=C6DF907289C366D83FFF28599FDA90FE.live11041?__blob=publicationFile&v=3). 

Should we use the NN suffix in this case, or sould this only be used if "global" nudging in the sense of the ICON tutorial page 160 is applied (= nuding in the complete domain)?

**Comment by jesusff** [comment]
I'd say that the idea of having the nudging setting promoted to the `source_id` using the suffixes -SN or -NN responded to the aim of informing users that these simulations are expected to follow more closely than others their driving fields. This is relevant to interpret the results. In this sense, the decision to add the suffix could be made based on the strength of this nudging and whether it is being applied to constrain the RCM to follow the driving fields, or just to prevent instabilities near the boundaries. For example, in your case, it might depend on the height of $z_{start}$ in Figure 6.1.

We have another example (#45), where @chunhsusu added the -NN suffix under these conditions (Section 2.3 in http://www.bom.gov.au/research/publications/researchreports/BRR-069.pdf):
> we use nudging based on Newtonian relaxation (Telford et al., 2008) to adjust dynamical variables of free running regional model towards the 6-hourly global model fields. The variables in the UM being nudged are horizontal wind components and potential temperature from model level 38 (11 km above surface) and above, with a relaxation factor of 1/6 h-1, based on Stassen et al. (2022).

In the cases where the nudging is not applied strongly to the interior of the domain, the decision to add the suffix or not is a bit subjective, but I think we should at least try to avoid having models registered without the suffix that applied a deeper nudging than those with the suffix.

**Comment by csteger** [comment]
I have read section 3.1 (Large scale nuding) here: http://www.bom.gov.au/research/publications/researchreports/BRR-069.pdf 

It sounds kind of similar to what we do, but we decided to switch the upper boundary nuding on mainly because we had some very few cases in test simulations in which the solution of the RCM deviated strongly from the driving data. 

In general I would say that SN or NN should be used if the nudging is applied in the whole domain. If nuding is only applied at the lateral and/or upper boundaries SN/NN should not be used because I assume that all groups do something here in one or another way. 

That would be a clear regulation in my opinion. Trying to distinguish between different levels of nudging at the boundaries and then use SN/NN or not is maybe more complicated.

**Comment by csteger** [comment]
How do we want to proceed here?

**Comment by jesusff** [comment]
@gnikulin is currently out of the office, but we can ping others for their opinion @larsbuntemeyer @sethmcg @jpevans

I'm not aware of that upper boundary relaxation to the driving fields in other models, but it could be a standard practice. In WRF, we usually apply a damping at the top levels to avoid reflections and instability, but no nudging to the large scale driving fields. Given that not so many models registered so far, we can try to have a quick survey on their treatment of the upper boundary (I changed the title of the issue, so they get a more appealing notification). Ping @YanjunJiao @chunhsusu @gredmond-mo  @pierrenabat @aulemahal @sdesrosiers1989 @Hanhle-CLIPSSA @graziano-giuliani

On the other hand, I would say that interior nudging is never applied to the whole domain, but from some level up to the top, as we need low level circulation free to interact with the surface. In a sense, this is quite similar to your upper boundary nudging, if the $z_{start}$ is sufficiently low.

In any case, the use of the suffixes is up to the groups. We can collect here the different configurations regarding the upper level boundary treatment, so the groups can make a more informed decision. These flags are just quick information for the data users. I cannot foresee whether the use of these flags in the `source_id` will lead to the simulations being under- or overused. I can imagine, though, future analyses trying to assess the consistency of the RCMs with their driving GCMs, and explaining the results simply by factorizing by the nudging flag, without needing to resort to further research on the individual model inner workings.

**Comment by sethmcg** [comment]
I believe that Newtonian nudging refers to nudging throughout the domain, not just relaxation of boundary conditions, but I am by no means an expert.

To expand on Jesus' point, one important use case is when you have two different simulations using the same RCM, but in one case it's nudged and the other it's not.  There are a number of simulations in the previous round of CORDEX like this, and it makes a big difference when you're analyzing and interpreting the results, so it's important to communicate that to end users.  The suffix doesn't need to contain all the information about nudging, it's more of a flag to signal that it's a factor to keep in mind.

**Comment by csteger** [comment]
Thanks for the feedback. At least for EURO-CORDEX we will use the same set up for all runs with standard configuration (atmosphere, land). There will be some simulations with coupled set ups that include ocean and runoff models, but they will use another source_id.

**Comment by larsbuntemeyer** [comment]
I have to admit, this is not really my area of expertise buit i would love to move this valuable content to [discuss](https://github.com/WCRP-CORDEX/discuss) and promote it to a proper [CORDEX discussion](https://github.com/orgs/WCRP-CORDEX/discussions) (if @gnikulin approves). I would then encourage @csteger to make up a new registration issue once this is resolved...

**Comment by csteger** [comment]
@jesusff @larsbuntemeyer 
Can we close this issue and register the source_id? I am not sure if there will be further feedback. We need the source_id for the CMORization of the runs in the near future. If possible I would go for ICON-CLM-202407-1-1

**Comment by jesusff** [comment]
Hi, Christian. Yes, but please could you open a separate issue to request again the registration? In this way, we can leave this for further discussion on the nudging issue

**Comment by csteger** [comment]
Okay. I will open a new issue.
